manim/mobject/svg/text_mobject.py

```diff
@@ -773,6 +773,7 @@ def __init__(
         if self.disable_ligatures:
             self.submobjects = [*self.gen_chars()]
         self.chars = VGroup(*self.submobjects)
+        self.words = VGroup(*self.gen_words())
```

Comment:
Why does `gen_words` return a VGroup just to be unpacked into a new VGroup? It seems to me that it may be best to just call gen_words here like
```suggestion
        self.gen_words()
```
and then alter `gen_words` below

It seems like you're copying how `self.chars` works, but I think this may be a better approach :)